features/ultils/world.ts

```diff
@@ -36,6 +60,40 @@ export class OurWorld extends World {
       destFilePath: filePath,
     });
   }
+
+  public getCredentialByAppKey(appKey: string): Credential {
+    const credential = getCredentialByAppKey(this.credentials, appKey);
+    if (credential === undefined) {
+      throw new Error(`The credential with app key ${appKey} is not found`);
+    }
+
+    if (!credential.appId) {
+      throw new Error(`The credential with app key ${appKey} has no App ID`);
+    }
+
+    return credential;
+  }
+
+  public getAPITokenByAppAndPermission(
+    appKey: string,
+    permissions: Permission[],
+  ): string {
+    const apiToken = getAPITokenByAppAndPermission(
```

Comment:
MUST:
From https://github.com/kintone/cli-kintone/pull/566#discussion_r1390522944
Please rename function names to `...Permissions`

```suggestion
  public getAPITokenByAppAndPermissions(
    appKey: string,
    permissions: Permission[],
  ): string {
    const apiToken = getAPITokenByAppAndPermissions(
```